add-version.sh

```diff
@@ -137,7 +136,7 @@ cask "galasactl" do
   url "https://github.com/galasa-dev/galasa/releases/download/v#{version}/galasactl-darwin-#{arch}",
       verified: "github.com/galasa-dev/galasa/releases/"
   name "Galasa Client"
-  desc "Client to interact with the Galasa ecosystem or local development environment. Latest version."
+  desc "Client to launch Galasa tests on a Galasa service or locally. Latest version"
```

Comment:
should we go fix up the existing desc fields to hav ethe same test ?

Reply:
Sure - done in https://github.com/galasa-dev/homebrew-tap/pull/9/commits/68337eabf2782129d5e839b5ae965f19e565773c